src/js/apps/patients/patients-main_app.js

```diff
@@ -34,7 +34,7 @@ export default RouterApp.extend({
 
     this.routeAction(defaultRoute, () => {
       defer(()=> {
-        this.replaceRoute(defaultRoute, defaultWorklist);
+        this.navigateRoute(defaultRoute, defaultWorklist);
```

Comment:
so we were actually swallowing this route in history.  The "default" even wasn't causing an issue with double-history